### PR TITLE
[DX-916] Release note 5.2.5

### PR DESCRIPTION
**Pull request by andyo-tyk**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link
https://deploy-preview-3805--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-gateway/release-notes/version-5.2/

https://deploy-preview-3805--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.2/
<br>

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md [hunk 1]
```diff
@@ -14,6 +14,42 @@ Minor releases are supported until our next minor or major release comes out. Th
 
 ---
 
+## 5.2.5 Release Notes 
+
+##### Release Date 7 Dec 2023
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md]:
```suggestion
##### Release Date 19 Dec 2023
```

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md [hunk 2]
```diff
@@ -14,6 +14,42 @@ Minor releases are supported until our next minor or major release comes out. Th
...
+
+#### Changelog {#Changelog-v5.2.5}
+
+#### Fixed
+<ul>
+ <li>
+ <details>
+ <summary>Long custom keys not maintained in distributed Data Planes</summary>
+
+ Fixed an issue where custom keys over 24 characters in length were deleted from the Data Plane Redis on key update in distributed (MDCB) setups.
```
Comment (dcs3spp) [inline tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md]:
```suggestion
Fixed an issue where custom keys over 24 characters in length were deleted from Redis in the Data Plane when key update action signalled in distributed (MDCB) setups.
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
PR LGTM. Requested minor suggestion to make changelog fix clearer

File: tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md (hunk 2 above)
Comment (andyo-tyk) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md]: "key update" is an action/signal from MDCB. Please check this proposed change with @tbuchaillot or @sredxny

Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md]: Thanks @andyo-tyk 

@tbuchaillot @sredxny ?

Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-gateway/release-notes/version-5.2.md]: looks good to me!

**Review by dcs3spp — APPROVED** [review]

**Comment by dcs3spp** [comment]
Before merging and releasing this need to add links to docker images for Gateway and Dashboard

**Pull request merged**